### ISSUE TITLE
typo in a request parameter when submitting doi

### ISSUE THREAD

**Issue by tloubrieu-jpl** [NASA-PDS/doi-ui#124]
## 🐛 Describe the bug

When we submit an updated DOI in the release form, there is a typo in the API request parameter `identifier`. `identifer` is used instead.

## 📜 To Reproduce
0. open the developer console in chrome
1. Select a draft DOI in the search screen
2. Click on the release button
3. Click on submit for review, check the ignore warning checkbox if necessary
4. The message says the submission is successful
5. But in the developer console, the url sent to the api is in error `http://localhost:8080/PDS_APIs/pds_doi_api/0.2/doi/submit?identifer=urn:nasa:pds:insight_cameras::5.0&force=true`

## 🕵️ Expected behavior

- [ ] - There should not be a success message, I am guessing only the error coming from the dois POST request are catch, we should catch error coming from the submit action as well.
- [ ] - The parameter of the query should be `identifier` instead of `identifer`

## 📚 Version of Software Used

Latest version of the UI on main branch, with latest version of the API (deployed on pds-dev3)

## 🖥 System Info

---

**Comment by eddiesarevalo** [comment]
@tloubrieu-jpl for the first item "There should not be a success message, I am guessing only the error coming from the dois POST request are catch, we should catch error coming from the submit action as well." 

For reserve there is a format for errors:
```
{
  "errors": [
    {
      "message": "There is already a DOI 10.13143/wj8z-df73 associated with PDS identifier urn:nasa:pds:insight_cameras::11.0 (status=review).\nYou cannot modify a DOI for an existing PDS identifier.",
      "name": "CriticalDOIException"
    }
  ]
}

```

For submit review there is no mention of error except for the status 400. 
```
{
  "detail": "Missing query parameter 'identifier'",
  "status": 400,
  "title": "Bad Request",
  "type": "about:blank"
}

```

Since it doesn't come as an error array it is not detect as an error. The only way to know would be to check the status number. Should I make it check for different status numbers or can you return an errors array as when reserving?

**Comment by tloubrieu-jpl** [comment]
@eddiesarevalo you are right, we don't need to develop something for this type of errors since it will not happen once the identifier parameter will not have the typo.

Lets forget about capturing the `submit` errors for now.
